mopidy/audio/actor.py

```diff
@@ -594,9 +597,16 @@ def set_uri(self, uri, live_stream=False):
         else:
             current_volume = None
 
+        flags = _GST_PLAY_FLAGS_AUDIO
+        if download:
+            flags |= _GST_PLAY_FLAGS_DOWNLOAD
+
```

Comment:
Or we could log the fact that both `download` and `live_stream` were True here.

Reply:
It doesn't see doesn't seem right to raise a warning for every track change, when it's more likely to be a problem with the design of the extension than with a track.

I was thinking there would be a place where I could do some like this, but I couldn't work out where:

```
for each backend:
   if backend.is_live() and backend.should_download():
      logger.warn("That doesn't make sense.")
```

Looking at the `is_live()` method now, it requires, the uri of a track as a parameter, so maybe it is a per-track issue, rather than a "you're extension is broken" one.

Reply:
I'll just put the logging as you suggest here for now.